composer.json

```diff
@@ -26,6 +26,7 @@
 		"bjeavons/zxcvbn-php": "^0.4.0",
 		"altis/browser-security": "^1.0.0",
 		"humanmade/hm-limit-login-attempts": "1.1.0",
+		"humanmade/php-basic-auth": "^1.1.0"
```

Comment:
Missing a comma here, this will error during `composer install` at the moment

Reply:
🤦‍♂